src/spx_fmt.c

```diff
@@ -190,21 +190,35 @@ void spx_fmt_row_print(const spx_fmt_row_t * row, spx_output_stream_t * output)
         ;
 
         if (row->cells[i].ansi_fmt) {
+#ifdef _WIN32
+#   pragma warning(push)
+#   pragma warning(disable: 4129)
+#endif
             spx_output_stream_printf(output, "\e[%sm", row->cells[i].ansi_fmt);
+#ifdef _WIN32
+#pragma warning(pop)
+#endif
```

Comment:
If you are sure that you want to disable this warning everywhere, you can add to config.w32:
````
ADD_FLAG("CFLAGS_SPX", "/wd4129");
````
and drop the `#ifdef`s from the C files.

Reply:
I don't think `\e` will work on Windows; maybe `\x1b` would, but that requires Windows 10 or a console with ANSI color support. Unless the colors are important, consider to introduce a conditionally defined macro, and use this instead.

Reply:
@cmb69 to be honest I haven't checked yet what is the actual support for ANSI sequences on windows. Given what you say I might disable color support (everything based on ANSI sequences) on windows.

Reply:
If it returns `true` does it mean that it will work the same way as on unix-like systems (with `\e[` as prefix) ?

Reply:
I think so, but `\e` is not supported on Windows; you would need to use `0x1b` or something like that (ASCII value of ESC),

Reply:
Ok :+1: